crypt/crypt.go

```diff
@@ -11,10 +11,15 @@ import (
 const (
 	initializationVectorSize = 16
 	integritySignatureSize   = 4
+	minimalMessageSize       = initializationVectorSize + integritySignatureSize
 )
 
 func DecryptHmacXorWithIntegrity(message, encryptKey, integrityKey []byte) ([]byte, error) {
 	size := len(message)
+	if size < minimalMessageSize {
+		return nil, fmt.Errorf("message length to short(<%d), message: %v, message-size: %d", minimalMessageSize, message, len(message))
```

Comment:
- typo "too short"
- when I read `message-size` I was thinking that this is the result of a subtraction with the `size` variable above, maybe you can replace "message-size:" with "length:" since it is the length of what was just printed before